sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -51,65 +51,10 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-sdk-bom</artifactId>
-        <version>1.2.4</version>   <!-- NOTE: This should be updated manually. -->
+        <version>1.2.5</version>   <!-- NOTE: This should be updated manually. -->
```

Comment:
Why it's 1.2.5, the latest version of this month should be 1.2.6 https://mvnrepository.com/artifact/com.azure/azure-sdk-bom